### PR TITLE
Update the unit tests to confirm that log messages actually get writt…

### PR DESCRIPTION
**Pull request by scarabus**
…en to the audit logs

Split out FTP and SFTP tests for Get and Send. Remove failing ychart test - was throwing Socket exception when sending PASV copmmand

### CODE REVIEW

**Comment by jeremy-flieder** [comment]
I see that the logging has changed but there's still no log message when can't connect to FTP server, it just says it couldn't retrieve the file

File: Libs/RaptureCore/src/main/java/rapture/kernel/AuditApiImpl.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
  * 
  */
 public class AuditApiImpl extends KernelBase implements AuditApi {
-    private static final RaptureURI KERNEL_URI = RaptureURI.builder(Scheme.LOG, "//kernel").build();
+    private static final RaptureURI KERNEL_URI = RaptureURI.builder(Scheme.LOG, "kernel").build();
```
Comment (jonathan-major) [inline Libs/RaptureCore/src/main/java/rapture/kernel/AuditApiImpl.java]: why the change?

File: Libs/WorkflowsCommonSteps/src/main/java/rapture/dp/invocable/ftp/steps/SendFileStep.java [hunk 2]
```diff
@@ -93,16 +94,20 @@ public String invoke(CallingContext ctx) {
...
             String retval = getNextTransition();
             int failCount = 0;
+            int successCount = 0;
             Connection connection = new SFTPConnection(configUri).setContext(ctx);
             List<FTPRequest> requests = new ArrayList<>();
             for (Entry<String, Object> e : map.entrySet()) {
                 String remote = e.getValue().toString();
                 // If the target is a URI then just use the leaf name. Eg blob://foo/bar/baz -> baz
-                if (remote.contains("://")) remote = new RaptureURI(remote, Scheme.DOCUMENT).getLeafName();
+                if (remote.contains("://")) {
```
Comment (jonathan-major) [inline Libs/WorkflowsCommonSteps/src/main/java/rapture/dp/invocable/ftp/steps/SendFileStep.java]: only ever handling documents here?

File: Libs/RaptureCore/src/main/java/rapture/kernel/AuditApiImpl.java (hunk 1 above)
Comment (scarabus) [inline Libs/RaptureCore/src/main/java/rapture/kernel/AuditApiImpl.java]: Because when I delved into the audit log I realized that it was creating entries for both log://kernel and log:///kernel

File: Libs/WorkflowsCommonSteps/src/main/java/rapture/dp/invocable/ftp/steps/SendFileStep.java (hunk 2 above)
Comment (scarabus) [inline Libs/WorkflowsCommonSteps/src/main/java/rapture/dp/invocable/ftp/steps/SendFileStep.java]: Only getting the leaf name so scheme doesn't matter

**Comment by scarabus** [comment]
Overhauled the FTP stuff. Added unit tests to verify that FTP errors are logged.
SFTP errors are harder to determine because the log is so noisy.
The 'run with retry' code wasn't working as intended so threw it out. (FTP either works or it doesn't anyway)

**Comment by scarabus** [comment]
Test this please
